StackExchange.Profiling/Timing.cs

```diff
@@ -81,7 +81,25 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public IList<Timing> Children
```

Comment:
Children now returns a copy of children, for 99% of the cases i think this is the best option